pages/rops/update/schema/index.js

```diff
@@ -140,7 +148,11 @@ module.exports = req => {
       format: toBoolean,
       automapReveals: true,
       options: [
-        false,
+        {
+          value: false,
+          disabled: disableProcRadio('endangered'),
+          warning: content.fields.disabledWarning
```

Comment:
Does the warning need to be conditional on the field being disabled?

Reply:
good suggestion.. done

Reply:
https://github.com/UKHomeOffice/react-components/blob/main/src/forms/radio-group.jsx#L72-L75

This should either be renamed to `disabledWarning` or it should always display the warning an the warning is only passed if needed, i.e.
```js
{
    opt.warning &&
    <div className="govuk-reveal">
        <Warning>{opt.warning}</Warning>
    </div>
}
```
I think I prefer the second option as it makes it possible to have warnings that always show.